modules/tensor_mechanics/include/materials/RadialReturnCreepStressUpdateBase.h

```diff
@@ -25,8 +25,12 @@ class RadialReturnCreepStressUpdateBase : public RadialReturnStressUpdate
 protected:
   virtual void initQpStatefulProperties() override;
   virtual void propagateQpStatefulProperties() override;
-  virtual void computeStressFinalize(const RankTwoTensor & plastic_strain_increment) override;
-
+  virtual void computeStressFinalizeMain(const RankTwoTensor & plastic_strain_increment,
```

Comment:
This will be the new version, right? You should just overload the name of the other one and name it `computeStressFinalize`, and then we'll get rid of the old one once we switch over the apps.

Reply:
Doing that causes an overload hiding error because all the derived classes should implement all or none of the methods with the same name (this appears to be in the language to avoid unintended consequences which should not apply here). Since we do not want to make changes to BISON, I believe changing the name is the option we have.